### PR TITLE
[INLONG-5165][Manager] Getting Sort source collects wrong topic properties of each stream

### LINKED ISSUE #5165: [Bug][Manager] Getting Sort source collects wrong topic properties of each stream
### What happened

The TopicProperties in SortSourceConfig, which is used to config sort sdk, collects wrong topic properties.
The correct topic properties of each stream should be maintain by ext_params in Stream_Sink , instead of Inlong_Group

### What you expected to happen

collect topic properties from ext_params in Stream_Sink

### How to reproduce

when try to config sort sdk

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #5165

### Motivation

The TopicProperties in SortSourceConfig, which is used to config sort sdk, collects wrong topic properties.
The correct topic properties of each stream should be maintain by ext_params in Stream_Sink , instead of Inlong_Group

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is already covered by existing tests, such as:
SortServiceImplTest

### Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? JavaDocs

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sortstandalone/SortSourceStreamInfo.java [hunk 1]
```diff
@@ -17,12 +17,38 @@
...
+    public Map<String, String> getExtParamsMap() {
+        if (extParamsMap != null) {
+            return extParamsMap;
+        }
+        if (StringUtils.isNotBlank(extParams)) {
+            try {
+                Gson gson = new Gson();
+                extParamsMap = gson.fromJson(extParams, Map.class);
+            } catch (Throwable t) {
+                LOGGER.error(t.getMessage(), t);
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sortstandalone/SortSourceStreamInfo.java:48]: Suggest setting more info in the error log, the exception stack trace has already included the message info.

File: inlong-sort/pom.xml [hunk 2]
```diff
@@ -40,7 +40,7 @@
         <module>sort-connectors</module>
         <module>sort-core</module>
         <module>sort-dist</module>
-        <module>sort-end-to-end-tests</module>
+        <!--<module>sort-end-to-end-tests</module>-->
```
Comment (healchow) [inline inlong-sort/pom.xml:43]: Please do not change this module, the unit tests error for this module will be fixed later.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-sort/pom.xml (hunk 2 above)
Comment (vernedeng) [reply inlong-sort/pom.xml:43]: sorry, foget to uncomment it.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sortstandalone/SortSourceStreamInfo.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sortstandalone/SortSourceStreamInfo.java]: fixed, thx

**Review by luchunliang — APPROVED** [review]
LGTM

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**